### DISCUSSION TITLE
Option to Sort Tabs & Groups in a Panel / Group

### DISCUSSION THREAD

**Discussion by JohnVKRDam** [mbnuqw/sidebery#1197] [Ideas]
As the Title says, an option to Sort Tabs & Groups in a Panel and/or Group, ascending / descending would be great.
The Sort option could be located in the right-click context menu for a Panel / Group?

**Comment by mbnuqw** [comment]
Issue to track: https://github.com/mbnuqw/sidebery/issues/643

**Reply by JohnVKRDam** [reply]
Oooh! I'm sorry! I only searched for existing questions about sorting in **_discussions_**, and mainly the **_Ideas_** section, because I didn't think of this as an issue, to be honest :)  But that is entirely on me, of course. Obviously any request for an improvement will need to be tracked, which is what the **_Issues_** section is for. Again, my apologies!

I will follow & track the aforementioned Issue. Thanks again! 👍
